premailer/premailer.py

```diff
@@ -85,8 +85,28 @@ def _cache_parse_css_string(css_body, validate=True):
     """
     return cssutils.parseString(css_body, validate=validate)
 
+
+def _capitalize_property(match):
+    return '{0}:{1}{2}'.format(
+        match.group('property').capitalize(),
+        match.group('value'),
+        match.group('terminator'))
+
+
+def _capitalize_float_margin(css_body):
```

Comment:
No need to mark this as private. It's not that secret :)

Also, the fact that you wanted to and were able (I love it!) write a test for this function, also tells me it doesn't need to be a "private" function.